pkg/tests/BUILD

```diff
@@ -12,9 +12,35 @@ genrule(
     cmd = "for i in $(OUTS); do echo 1 >$$i; done",
 )
 
+pkg_tar(
+    name = "test-tar-package-dir",
+    srcs = [
+        ":etc/nsswitch.conf",
+    ],
+    package_dir = "/package"
+)
+
+genrule(
```

Comment:
Why don't you just check in the file  'testdata/package_dir.txt' with the content '/package'.  That will avoid Windows compatibility problems with the genrule.  Then you also do not need to include this in the archive_testdata rule, because the glob will get it.

Reply:
@aiuto good point! addressed in https://github.com/bazelbuild/rules_pkg/pull/57/commits/fa82eea821e9cadd65cc994215e8c7fdb08c5284
I haven't really considered that Windows is incompatible with UNIX `echo`